documentation/tut/toy-example-reporter.rst

```diff
@@ -0,0 +1,241 @@
+
+Under the hood, this command is adding the following entities:
+    - A yearly sensor that stores the capacity of the grid connection.
+    - A power sensor, `headroom`, to store the remaining capacity for the battery. This is where we'll store the report.
+    - A `ProfitLossReporter` configured to use the prices that we setup in Tut. Part II.
+    - Three sensors to register the profit of running the process of Tut. Part III.
+
+Let's check it out! 
+
+Run the command below to show the values for the `Grid Connection Capacity`:
+
+.. code-block:: bash
+
+    $ TOMORROW=$(date --date="next day" '+%Y-%m-%d')
+    $ flexmeasures show beliefs --sensor-id 7 --start ${TOMORROW}T00:00:00+02:00 --duration PT24H --resolution PT1H
+      
+      Beliefs for Sensor 'Grid Connection Capacity' (ID 7).
+        Data spans a day and starts at 2023-08-14 00:00:00+02:00.
+        The time resolution (x-axis) is an hour.
+        ┌────────────────────────────────────────────────────────────┐
+        │                                                            │ 
+        │                                                            │ 
+        │                                                            │ 
+        │                                                            │ 
+        │                                                            │ 1.0MW
+        │                                                            │ 
+        │                                                            │ 
+        │                                                            │ 
+        │▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀│ 0.5MW
+        │                                                            │ 
+        │                                                            │ 
+        │                                                            │ 
+        │▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁│ 0.0MW
+        │                                                            │ 
+        │                                                            │ 
+        │                                                            │ 
+        │                                                            │ -0.5MW
+        └────────────────────────────────────────────────────────────┘
+                5            10            15           20
+                        ██ Grid Connection Capacity
+
+
+Moreover, we can check the freshly created source `<Source id=6>` which defines the `ProfitLossReporter` with the required configuration. You'll notice that the `config` is under `data_generator` field. That's because reporters belong to a bigger category of classes that also contains the `Schedulers` and `Forecasters`.
+
+.. code-block:: bash
+
+    $ flexmeasures show data-sources --show-attributes --id 5
+
+         ID  Name          Type      User ID    Model           Version    Attributes                                  
```

Comment:
I think `Type` is now a header and no longer a column.

Reply:
Is this resolved? Probably it indeed makes more sense to have the type as one of the columns in the case that a single source is requested. When I currently run this command, it still shows the type as a separate header above the table.